### DISCUSSION TITLE
unable to start nginx service

### DISCUSSION THREAD

**Discussion by aambadkar** [learndevopsonline/learndevopsonline#1670] [Q&A] [answered]
hello,

when i run role in ansible for frontend its says unable to start nginx service because control process exited

![image](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/db589d33-08b4-4205-9f1b-74f36557b18a)

**Comment by sreedharm07** [comment]
@aambadkar 
in third line double quotes no needed also 8080 port missing in same line
check service file from doc again and give properly
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/a2dedd8f-d60e-4892-badc-afc6df7a1153)

**Reply by aambadkar** [reply]
my file is this one 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/ea5bd227-0c1d-464f-bda7-b13c8a434e3a)

**Reply by sreedharm07** [reply]
ya what i attached also same 
but i seen from server ( applied parameters in server )

**Reply by sreedharm07** [reply]
also your backend server was not in up

**Reply by aambadkar** [reply]
i updated

still having same error
![image](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/86959b13-2d56-4078-82af-9f46fd067f19)

**Reply by sreedharm07** [reply, marked as answer]
your domain name doesnt exists
check route 53 records properly i think domain name was wrong you given
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/f9032cbf-8193-4fa6-895a-46efb6d54290)
